net-im/swiften/metadata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		Swiften is a robust, high-quality, standards-compliant,
+		cross-platform, and performant C++ library for
+		implementing XMPP applications.
+	</longdescription>
+	<use>
+		<flag name="debug">Compile with debug information.</flag>
```

Comment:
Please make this more explanatory. 'Debug information' makes me think of `-g`.

Reply:
Do I have to write more, if the default global use flag description would be find? Basically, debug enables the output of debug messages, which most classes of swiften seems to do.

Reply:
Yes. Prefer no unnecessary duplication.

Reply:
Thanks, pushed now.